main.py

```diff
@@ -881,6 +891,21 @@ def parse_args(self) -> None:
             self.parse_configuration(configuration["config"], args.targetURL)
         else:
             self.parse_configuration(vars(args), args.targetURL)
+    
+    def print_help_formatted(self, parser):
+        print("| Argument | Description |")
+        print("| --- | --- |")
+
+        argsPerHelpText = []
+        lastHelpText = list(parser._option_string_actions.values())[0]
+        for arg, helpText in parser._option_string_actions.items():
+            if(lastHelpText != helpText):
+                descriptionText = lastHelpText.help.replace('\n', '<br>')
+                print(f"| {', '.join(argsPerHelpText)} | {descriptionText} |")
+                argsPerHelpText.clear()
+
+            argsPerHelpText.append(arg)
+            lastHelpText = helpText
```

Comment:
sorry, the code I gave you omitted the last argument (and there's a better way to do it)

```suggestion
def print_help_formatted(self, parser):
    print("| Argument | Description |")
    print("| --- | --- |")

    allRows = []
    for arg, helpText in parser._option_string_actions.items():
        descriptionText = helpText.help.replace('\n', '<br>')
        allRows.append(f"| {', '.join(helpText.option_strings)} | {descriptionText} |")

    # remove duplicates
    for row in list(OrderedDict.fromkeys(allRows)):
        print(row)
```

you'll need to import OrderedDict too
```
from collections import OrderedDict
```